Source/santad/SNTExecutionController.mm

```diff
@@ -92,6 +93,7 @@ @implementation SNTExecutionController {
   absl::Mutex _entitlementFilterMutex;
   std::set<std::string> _entitlementsTeamIDFilter;
   std::unique_ptr<PrefixTree<Unit>> _entitlementsPrefixFilter;
+  std::unique_ptr<SantaCache<pid_t, bool>> _procSignalCache;
```

Comment:
We should really make sure to use pid/pidver pair here. The concern is a bit far fetched, but there are a few paths through where a SIGSTOPped process could exit before an XPC reply to cleanup (either due to connection being invalidated, or one of the few ways through the kernel where a signal can be sent without hitting the MAC hook). If a PID is recycled, we would easily get into a weird state.

Reply:
Fair enough, I've switched to using a std::pair to hold both. This necessitated some changes to SantaCache - I could have added a SantaCacheHasher specialization but I've switched over to using absl::Hash instead as that handles most things we want to hash and almost certainly does a better job than what I came up with several years ago.

Reply:
Love this change to `absl::Hash`!